.github/workflows/prepare-for-prod-dt-deploy.yml

```diff
@@ -144,3 +144,39 @@ jobs:
           git push -u origin hosted-transformer-$TAG_NAME
 
           gh pr create --fill
+
+      - name: Update helm charts and raise pull request for enterprise customers on dedicated transformers
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          cd rudder-devops
+          git checkout -b dedicated-transformer-$TAG_NAME
```

Comment:
In most of the cases, we have dedicated transformer with feature branch deployment rather than regular version. So need to be careful in reviewing automatic PR being raised by this action.